vignettes/Surveys_Walkthrough.Rmd

```diff
@@ -22,9 +22,7 @@ knitr::opts_chunk$set(
 
 ## Overview
 
-This page demonstrate use of the {sitrep} R Markdown template for analysis of a retrospective mortality survey, using data from TODO. *This case study uses data that was produced using Kobo, a common survey and data collection tool in settings where MSF works.*
-
-### Survey background
+This page demonstrate use of the {sitrep} R Markdown template for analysis of a retrospective mortality survey, using data that was produced using Kobo, a common survey and data collection tool in settings where MSF works.
```

Comment:
mention that is is not the standard msf erb pre-approved dictionary?

Reply:
done